.github/workflows/rust.yml

```diff
@@ -145,6 +149,10 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: true
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v2
+        with:
+          version: "23.2"
```

Comment:
Where does the protoc compiler get introduced? Is this from `celestia-types`?

Reply:
It's a dependency of our other pkg, `celestia-proto`.

Reply:
It's used to build MsgPayForBlobs and other celestia and cosmos relevant protos

Reply:
Makes sense, thanks!